junit-runtime/src/main/scala/org/scalajs/junit/JUnitExecuteTest.scala

```diff
@@ -199,8 +199,12 @@ final class JUnitExecuteTest(taskDef: TaskDef, runner: JUnitBaseRunner,
 
   private[this] def logFormattedInfo(method: String, msg: String): Unit = {
     val fMethod = if (method != null) c(method, NNAME2) else null
-    richLogger.info(
-        formatLayout("Test ", packageName, c(className, NNAME1), fMethod, msg))
+    val formatted = formatLayout("Test ", packageName, c(className, NNAME1),
+        fMethod, msg)
+    if (verbose)
+      richLogger.info(formatted)
+    else
+      richLogger.debug(formatted)
```

Comment:
Hum... That now makes everything that goes to info go to debug if `-v` is not turned on. Are you sure this is intended? The documentation of `-v` says:

> Log "test run started" / "test started" / "test run finished" events on log level "info" instead of "debug".